Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,10 +1,35 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
 
 namespace Logger.Tests
 {
     [TestClass]
     public class FileLoggerTests
     {
+        [TestMethod]
+        public void FileLogger_AppendsLogsToFile()
+        {
+            string message1 = "What is the average air speed velocity of an unladen swallow?";
+            string message2 = "African or European?";
+
+            if(File.Exists("fileLoggerTest.txt"))
+                File.Delete("fileLoggerTest.txt");
```

Comment:
```suggestion
                File.Delete(fileName);
```

An even better suggestion would be to use the `GetRandomFileName()` method to generate a file name and rather than cleaning up after a previous unit test - you can delete the file at the end of the test that way you don't have extra data lying around after a test has finished running. (Just be sure to wrap the `File.Delete(...)` method in a finally block so it gets removed even if the unit test fails.)